packages/wonder-blocks-i18n/src/functions/get-locale.js

```diff
@@ -0,0 +1,13 @@
+// @flow
+const HEADERS_GLOBAL = "__HEADERS__";
```

Comment:
I'll admit that this scares me a bit. I know that this will work with our current system, but it definitely feels fragile. I'd love for us to have a way to work around this - maybe something we can set on initialization?

Reply:
It's super fragile.  Calling some initialization function feels fragile too but maybe there's some way we can guarantee that it's being called before anything else by calling it one of the page building functions.